PDSA_System/Server/Controllers/BrukerController.cs

```diff
@@ -71,14 +87,14 @@ public async Task<ActionResult<List<Bruker>>> CreateBruker(Bruker bruker)
         /*
          Updater en Bruker --> ikke helt funksjonell enda.
          */
-        [HttpPut]
+        [HttpPut("/api[Controller]/admin/UpdateBruker")]
         public async Task<ActionResult<List<Bruker>>> UpdateBruker(Bruker bruker)
         {
             var connString = _configuration.GetValue<string>("ConnectionStrings:DefaultConnection");
             using var conn = new DbHelper(connString).Connection;
 
             await conn.ExecuteAsync(
-                "UPDATE Bruker SET ForNavn = @ForNavn, EtterNavn = @EtterNavn, Email = @Email, PassordHash = @PassordHash, WHERE BrukerId = @BrukerId",
+                "UPDATE Bruker SET ForNavn = @ForNavn, EtterNavn = @EtterNavn, Email = @Email, PassordHash = @PassordHash, SjefsId = @SjefsId, WHERE BrukerId = @BrukerId",
```

Comment:
Kanskje greit å kalle den for `SjefID` istedenfor `SjefsID`? Fordi vi bruker jo `BrukerID` og **ikke** `BrukersID`

Reply:
Går vel mer på at BrukerId refereres til egen, mens SjefsID refererer til din sjefsId

Reply:
Men vi kan godt endre det, dersom det er ønskelig

Reply:
Står Sjef for Teamleder i dette tilfellet?

Reply:
Tror det er best å endre dette, er mye enklere å glemme bort en ekstra `s` enn å huske den

Reply:
ok, i fix

Reply:
@tbhansen ikke nødvendigvis, en teamleder vil også kunne ha en SjefId

Reply:
@tbhansen  ok